packages/core/fixtures/driver-test-suite/conn-liveness.ts

```diff
@@ -0,0 +1,49 @@
+import { actor } from "@rivetkit/core";
+
+export const connLivenessActor = actor({
+	onAuth: () => {},
+	state: {
+		counter: 0,
+	},
+	options: {
+		lifecycle: {
+			connectionLivenessInterval: 5_000,
+			connectionLivenessTimeout: 5_000,
+		},
+	},
+	actions: {
+		getActiveWsConnectionsCount: (c) => {
+			return Array.from(c.conns.values()).filter(
+				(conn) => conn.__persist.d === "genericWebSocket",
```

Comment:
`__persist` should be private. this was before i knew about the `export const xxxx = Symbol()` trick.

let's not use this in tests since it _should_ be private.

these tests should be based off of public properties.